README.md

```diff
@@ -32,12 +32,18 @@ Q2: What is the answer?
 Hello!  In the rush that is the start of the term, I really haven't had a chance to chat with many of you.  So would you do me the favor of helping me get to know a bit about you by answering the following?  Please avoid one word answers... I want to learn about you!
 
 Q1: So you're finishing up your time in this CSCC program.  Why did you decide to embark on this journey?
+A1: Having no computer science background, I was recommended to take the course to gain some technical acumen.
 
 Q2: I _love_ to travel.  Tell me a bit about your favorite travel destination so far.  What's on your travel bucket list?
+A2: I also love travel, and so does my wife.  We've been to lots of places in the states, but have yet to travel abroad, which we hope to remedy next year.  So far, I've loved Colorado, Chicago, Vegas, and Disney World best of all.
```

Comment:
Awesome!  So where are you planning/hoping to travel abroad?  Please add that detail and update your pull request.